build.gradle

```diff
@@ -1,27 +1,37 @@
 plugins {
     id 'java'
     id 'maven-publish'
+    id "net.kyori.blossom" version "1.1.0"
 }
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.8
+targetCompatibility = 1.8
 
-version = 'local'
+version = 'fabric-SNAPSHOT-0.0.2' // SNAPSHOT triggers commit hash info
```

Comment:
this isnt really a valid version, id keep this as 0.0.2 and move the info else where tbh. IDK what you think?

Reply:
Sure, I can patch the template java file instead. Wait for me